### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by Federerer** [dsharlet/LiveSPICE#137] [Show and tell]
Hi guys, how do you feel about some UI refresh? I've been thinking about adding some new features to the VST plugin (status bar, automatic schematic reloading, etc), and I've been playing also with some UI layout ideas. Here is a quick mockup: 
![ui](https://user-images.githubusercontent.com/19556976/134743617-e722e68a-7bec-45ab-acbb-54ad1053d489.png)
It's definitely not finished, and some components like those `ComboBoxes` are not styled at all. I just wanted to get your initial opinion if it's worth to continue this effort? 😃

You can check the sources [here](https://github.com/Federerer/LiveSPICE/tree/feature/vst_ui).

**Comment by dsharlet** [comment]
I think it looks nice :) @mikeoliphant is the primary developer of the VST plugin, he might have thoughts.

**Comment by mikeoliphant** [comment]
Looks fine to me - although the schematic is taking up a lot of space for an element that isn't actually functional in the UI. I haven't been actively using this stuff for a while, though, so I don't really have a strong opinion...

**Comment by Federerer** [comment]
Quick update how it looks now. I've also added a ComboBox to quickly switch between files in the current folder 😄
![LiveSPICEVst](https://user-images.githubusercontent.com/19556976/137036860-d03d652b-5686-4c47-bd52-8f4668da7472.gif)
